checkbox_input.sh

```diff
@@ -119,11 +121,13 @@ remove_checkbox_instructions() {
 }
 
 checkbox_input() {
-  prompt=$1
+  local variable=$1
+  shift
+  local prompt=$1
   eval list=( '"${'${2}'[@]}"' )
-  _current_index=0
-  _first_keystroke=true
-
+  local _current_index=0
```

Comment:
`_current_index` seems to be used in other functions, so having a local scope to this variable means other functions might not be able to access it http://tldp.org/LDP/abs/html/localvar.html